### DISCUSSION TITLE
<circRNA>

### DISCUSSION THREAD

**Discussion by RavitaHooda** [community/community#43468] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I wanted to know how can we analyze circRNA data using a BAM file or fastaq file. 
How to analyze in  circompara using fastaq file format and covert fastaq file format into comma text seperated file or csv file.
or any other way of analysing circRNA data from BAM or Fastaq files.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @RavitaHooda! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by RavitaHooda** [comment]
Thank you
